js/libs/builder.js

```diff
@@ -750,56 +750,6 @@ new Vue({
         this.accessRules.splice(accessRuleIndex, 1);
       }
     },
-    setupCodeEditor: function() {
```

Comment:
This looks like we're removing code editor from somewhere else in the form builder that's unrelated to the Rich Text field. It looks like the form submission confirmation HTML. We can't remove that.

What version of TinyMCE is running in the builder interface? If it's TinyMCE 4, then we can keep this running without having to change anything, right?

Reply:
I returned setup for builder.